bindings/wasm/examples/worker.ts

```diff
@@ -457,8 +460,28 @@ async function exportGLB(manifold?: Manifold) {
     wrapper.addChild(node);
   }
 
+  const results: {glbURL?: string, threeMFURL?: string} = {};
+  if (manifold != null) {
+    const mesh = manifold.getMesh();
+    let vertices = new Float32Array(mesh.numVert * 3);
+    for (let i = 0; i < mesh.numVert; ++i) {
+      for (let j = 0; j < 3; ++j)
+        vertices[i * 3 + j] = mesh.vertProperties[i * mesh.numProp + j];
+    }
+    const model =
+        to3dmodel({simple: [{vertices, indices: mesh.triVerts, id: '1'}]});
+    const files: Zippable = {};
+    files['3D/3dmodel.model'] = strToU8(model);
+    files[fileForContentTypes.name] = strToU8(fileForContentTypes.content);
+    const zipFile = zipSync(files);
+    results['threeMFURL'] = URL.createObjectURL(new Blob(
```

Comment:
I don't think there's any way around this for `files`, but let's prefer e.g. `results.threeMFURL`.

Reply:
sure, I forgot to change this previously